pkg/guestagent/api/const.go

```diff
@@ -0,0 +1,12 @@
+package api
+
+import (
+	"net"
+	"strconv"
+)
+
+var IPv4loopback1 = net.IPv4(127, 0, 0, 1)
```

Comment:
Technically this is not a const, although I get the point.

Reply:
Removed this from here and moved to limayaml (This is where its actually used)